### DISCUSSION TITLE
Can i run this in Windows 10/11?

### DISCUSSION THREAD

**Discussion by MiruiWang2021** [cyring/CoreFreq#434] [Q&A] [answered]
Can i run this in Windows 10/11?

**Comment by cyring** [comment, marked as answer]
> Can i run this in Windows 10/11?

No. The Windows kernel was not retained due to its multiple constraints in advanced driver development.

For its part, the Linux kernel is a real pleasure for low-level programming: 
* open-source and shared knowledge 
* optional kernel security restriction 
* lightweight development environment
* Ease of network functions (ssh)
* An obvious choice in high performance computing
